nextflow.config

```diff
@@ -190,9 +191,11 @@ profiles {
         apptainer.enabled       = false
         docker.runOptions       = '-u $(id -u):$(id -g)'
     }
-    arm {
+    apple {
```

Comment:
this is a breaking change -> rnaseq 4.0.0?

Reply:
Just trying something, we can revert it!

Reply:
https://github.com/nf-core/rnaseq/issues/1497#issue-2817069206

Reply:
@mashehu I think the new arm profile(just setting the profile process.arch) is the same functionality, and maybe just an improvement. So, just a minor release is needed.